client/a8c-for-agencies/sections/purchases/licenses/license-preview/use-license-actions.ts

```diff
@@ -0,0 +1,85 @@
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import getLicenseState from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-license-state';
+import {
+	LicenseState,
+	LicenseAction,
+	LicenseType,
+} from 'calypso/jetpack-cloud/sections/partner-portal/types';
+import { urlToSlug } from 'calypso/lib/url/http-utils';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+export default function useLicenseActions(
+	siteUrl: string | null,
+	attachedAt: string | null,
+	revokedAt: string | null,
+	licenseType: LicenseType,
+	isChildLicense?: boolean
+): LicenseAction[] {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	return useMemo( () => {
+		if ( ! siteUrl ) {
+			return [];
+		}
+
+		const siteSlug = urlToSlug( siteUrl );
+
+		const handleClickMenuItem = ( eventName: string ) => {
+			dispatch( recordTracksEvent( eventName ) );
+		};
+
+		const licenseState = getLicenseState( attachedAt, revokedAt );
+		return [
+			{
+				name: translate( 'Set up site' ),
+				href: `https://wordpress.com/home/${ siteSlug }`,
```

Comment:
Can we set all the href to `""` for now and add a `FIXME:` comment? The reason is that we might miss updating this before the launch. Searching for "FIXME:" or "TODO:" throughout the project will help us find the missing items and update them.

Reply:
Make sense 👍